content/en/docs/tasks/security/authn-policy/index.md

```diff
@@ -19,6 +19,11 @@ the underlying concepts in the [authentication overview](/docs/concepts/security
 [installation steps](/docs/setup/install/kubernetes/#installation-steps), or set `global.mtls.enabled` to false using
 [Helm](/docs/setup/install/helm/)).
 
+{{< warning >}}
+If multiple different trust domains are used in your mesh, you should turn off the trust domain validation before
```

Comment:
Is this meant to be different or multiple? Thanks!

Reply:
different. Updated to make the text more clear.